assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+            "en": "internet access price"
+          },
+          "question": {
+            "en": "Do you have to pay for the internet access?"
+          },
+          "freeform": {
+            "key": ""
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "internet_access:fee=yes"
+                ]
+              },
+              "then": {
+                "en": "You need to pay extra for internet access"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "internet_access:fee=no"
+                ]
+              },
+              "then": {
+                "en": "You  do not need to pay extra for internet access"
+              }
+            }
+          ],
+          "condition": {
+            "and": [
+              "internet_access=yes"
+            ]
+          }
+        },
+        "reviews",
+        {
+          "render": {
+            "en": "toilets"
+          },
+          "question": {
+            "en": "Does this place have toilets?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "toilets=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has toilets"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "toilets=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have toilets"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "toilets"
+          }
+        },
+        {
+          "render": {
+            "en": "More details about this place: {description}"
+          },
+          "question": {
+            "en": "Would you like to add a general description of this place? (please keep it objective; opinions go into the reviews)"
+          },
+          "freeform": {
+            "key": "description",
+            "type": "text"
+          }
+        }
+      ],
+      "hideUnderlayingFeaturesMinPercentage": 0,
+      "icon": {
+        "render": "./assets/themes/campersites/caravan.svg",
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=no"
+              ]
+            },
+            "then": {
+              "en": "./assets/themes/campersites/caravan_green.svg"
+            }
+          }
+        ]
+      },
+      "width": {
+        "render": "8"
+      },
+      "iconSize": {
+        "render": "40,40,center",
+        "mappings": [
+          {
+            "if": {
+              "and": [
+                "fee=yes"
+              ]
+            },
+            "then": {
+              "en": "This place charges a fee"
+            }
+          },
+          {
+            "if": {
+              "and": [
+                "fee=no"
```

Comment:
Did you copy-paste from the tagrenderings here? You are marking icon sizes here...

Reply:
No idea how this happened